pyproject.toml

```diff
@@ -28,4 +28,4 @@ flake8 = "*"
 twine = "*"
 wheel = "*"
 isort = "*"
-black = {python=">3.6", version=">=18.9b0", allow_prereleases=true}
+black = {python=">3.6", version="^18.9b0", allow-prereleases=true}
```

Comment:
These days poetry seems to install `black` fine without the `allow-pre` parameter at all (I guess it derives it from the version spec?)

Reply:
OK, I'm happy to change this if you'd like me to?

Reply:
Yeah cool, might as well remove it